foursight_core/s3_connection.py

```diff
@@ -5,6 +5,7 @@
 import logging
 from foursight_core.abstract_connection import AbstractConnection
 from dcicutils.misc_utils import full_class_name
+from .boto_s3 import boto_s3_client, boto_s3_resource
```

Comment:
I'm assuming you're busy backing out of this.

Reply:
Yes